packages/modular-site/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
this is required because of a of the existence of multiple versions of babel-jest. will fix.